agent_factory/integrations/telegram/fieldeye_handlers.py

```
@@ -0,0 +1,598 @@
+                'codec': metadata.codec,
+                'resolution': f"{metadata.width}x{metadata.height}",
+                'fps': metadata.fps
+            }
+        }
+
+        result = storage.client.table("field_eye_sessions").insert(session_data).execute()
+        session_id = result.data[0]['id'] if result.data else None
+
+        # Format response
+        response = "✅ *Video Processed Successfully!*\n\n"
+        response += f"📊 *Summary:*\n"
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)